reporting/disconnect.html

```diff
@@ -9,20 +9,18 @@
 <script>
 promise_test(async test => {
     let observer;
-    let observerPromise = new Promise(resolve => {
+    const observerPromise = new Promise(resolve => {
```

Comment:
```suggestion
    const reportsPromise = new Promise(resolve => {
```

Naming it by what it resolves with.